### ISSUE TITLE
Tag version 9.0.0

### ISSUE THREAD

**Issue by Jamie-SA** [semanticarts/gist#140]
I think the original 9.0.0 release is this:
https://github.com/semanticarts/gist/tree/f7d442c47a4db3ccd5789ccc55b9104e6ac0280b

But MarkW made a change to the catalog file on July 26, 2019 (see #73) that I think was put out onto the website. It isn't really part of the Ontology... should it get tagged as the release? Or is that 9.0.1?
It was merged into master here:
https://github.com/semanticarts/gist/tree/e612a546922cb8986522c2a21ec537b945a125de

**Comment by Jamie-SA** [comment]
Summary of what I did in case anyone wants to comment on them (or we add them to a release script).
```
git checkout f7d442c47a4db3ccd5789ccc55b9104e6ac0280b
export GIT_COMMITTER_DATE="$(git show --format=%aD | head -1)"
git tag -a v9.0.0 -m "Release v9.0.0"
git push origin v9.0.0
```

**Comment by uscholdm** [comment]
Jamie said: 
> It isn't really part of the Ontology... should it get tagged as the release? Or is that 9.0.1?

The catalog file should be included in the release.  Having it be 9.0.1 seems a bad idea.

**Comment by Jamie-SA** [comment]
@uscholdm the catalog file is in the release. However, the problem is that there was a 9.0.0 release that had a catalog that had problems. There was a change after that to fix those issues. That is now what is on the website. They are not the same.

Any change in the ontology would result in a different version number. But in this case you could argue that 1) the catalog file is part of the ontology and therefore should increment the version number, or 2) that it is not part of the ontology and therefore doesn't need a version number change.

However, they are different commits in github. The official first release is not the same as the current release. So you have to decide which one should be tagged with v9.0.0 and the first one should have been tagged when it was released.

**Comment by marksem** [comment]
Probably should label fixed catalog version as 9.0.1.  That way, as Jamie states, you would know there was a "fixed" version past the original download version.

**Comment by rjyounes** [comment]
DONE
